src/marker-index.coffee

```diff
@@ -381,12 +381,16 @@ class MarkerIndex
       Range(start, @getEnd(id))
 
   getStart: (id) ->
-    if entry = @rangeCache[id]
-      entry.start ?= @rootNode.getStart(id)
+    return unless @rootNode.ids.has(id)
```

Comment:
I think that whereas before, we could use the id's presence in the `rangeCache` to know that the marker existed, we now create the `rangeCache` entries lazily, so we have to check the `rootNode` to know if the marker exists.